includes/polling/ports/brocade.inc.php

```diff
@@ -0,0 +1,11 @@
+<?php
+
+$brocade_stats = snmpwalk_group($device, 'swFCPortName', 'SW-MIB', 1, $brocade_stats);
+
+$brocade_ports = [];
+foreach ($brocade_stats as $index => $port) {
+    $index_brocade=$index + 1073741823;
+    $brocade_ports[$index_brocade]['ifAlias'] = $brocade_stats[$index]['swFCPortName'];
```

Comment:
Instead of making a temp variable, just set ifAlias into $port_stats.

Reply:
Thanks, removed the temp variable and set $port_stats